### PR TITLE
[Excel] (Custom functions) Naming updates

### PR DESCRIPTION
**Pull request by mscharlock**
Misc recommendations for naming page per Linda's suggestions a while ago

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[8933d19](https://github.com/OfficeDev/office-js-docs-pr/pull/930/commits/8933d19303fdba750d7b20a3c14d5d8929f5b854)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-naming.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-namingupdates/docs/excel/custom-functions-naming.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C29%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904290141227285-930%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=fRiBuWQkqCnYqi4DSfE5%2FqZ3S7M7DookwpZpa6cFNUQ%3D&st=2019-04-29T01%3A37%3A27Z&se=2019-05-30T01%3A42%3A27Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[c258e79](https://github.com/OfficeDev/office-js-docs-pr/pull/930/commits/c258e79ec2a62ac7b0d48b2aa51dc513e8475e47)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-naming.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-namingupdates/docs/excel/custom-functions-naming.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C29%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904290142597113-930%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=G4OxlmdHThsbWCWOL5Z7u9stB5pCLZysv9xw%2FRVySmI%3D&st=2019-04-29T01%3A39%3A05Z&se=2019-05-30T01%3A44%3A05Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/custom-functions-naming.md [hunk 1]
```diff
@@ -30,7 +30,12 @@ If your function name is the same as a function name in an add-in that already e
 
 To fix a name conflict, change the name in your add-in and try the function again. You can also uninstall the add-in with the conflicting name. Or, if you're testing your add-in in different environments, try using a different namespace to differentiate your function (such as NAMESPACE_NAMEOFFUNCTION).
 
-Also consider how you'd like people to use the functions within your add-in. In many cases, it makes sense to add multiple arguments to a function rather than create multiple functions with the same or similar names.
+## Best practices
+
+-  Consider adding multiple arguments to a function rather than create multiple functions with the same or similar names.
```
Comment (AlexJerabek) [inline docs/excel/custom-functions-naming.md]: Nit: extra space at the start of the line.

Comment (AlexJerabek) [inline docs/excel/custom-functions-naming.md]: "than create" -> "than creating"

**Comment by o365devx** [comment]
Docs Build status updates of commit _[45e670d](https://github.com/OfficeDev/office-js-docs-pr/pull/930/commits/45e670d143d5e3ec3edb85b962a23c157f5018ab)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-naming.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-namingupdates/docs/excel/custom-functions-naming.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C29%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904291818460180-930%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=DFYDpii1HD5%2Bf2NUQDde1rsTaoTWzfLUiEjPa2f3df4%3D&st=2019-04-29T18%3A15%3A01Z&se=2019-05-30T18%3A20%3A01Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**